Sources/FluentUI_iOS/Components/Command Bar/CommandBarButton.swift

```diff
@@ -90,6 +88,9 @@ class CommandBarButton: UIButton {
         isSelected = isPersistSelection && item.isSelected
         isHidden = item.isHidden
 
+        configuration?.title = item.title
+        configuration?.image = item.iconImage
```

Comment:
Should this go below the guard check on line 95 or is it fine to set these if there is `customControlView`?